integration-tests/src/test/scala/play/libs/ws/ahc/AhcWSClientSpec.scala

```diff
@@ -99,10 +99,11 @@ class AhcWSClientSpec(implicit executionEnv: ExecutionEnv) extends Specification
         val responseXml = response.getBody(xml())
         responseXml.normalizeDocument()
 
-        responseXml.isEqualNode(document) must beTrue
+        responseXml.isEqualNode(document) must beTrue and {
```

Comment:
Hi @cchantep, sorry, I don't get your point about why the files above aren't a good place to test `getUri`/`uri` methods and why do you think here is a better place.